public/scripts/Scene.js

```diff
@@ -0,0 +1,13 @@
+export class Scene {
+  constructor(data) {
+
+  }
+
+  switchTo() {
+    console.log('Scene switchTo not implemented!');
+  }
+
+  switchFrom() {
+    console.log('Scene switchFrom not implemented!');
+  }
```

Comment:
Probably remove the console log, thought this wasn't implemented yet. But the different scenes are overriding this. Maybe put a comment above to say it'll be overrode

Reply:
sure i can add a comment